src/sentry/api/endpoints/artifact_bundles.py

```diff
@@ -0,0 +1,110 @@
+from django.db import router
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint, ProjectReleasePermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize
+from sentry.models import ArtifactBundle, ProjectArtifactBundle
+from sentry.utils.db import atomic_transaction
+
+
+@region_silo_endpoint
+class ArtifactBundlesEndpoint(ProjectEndpoint):
+    permission_classes = (ProjectReleasePermission,)
```

Comment:
@mitsuhiko which permissions do we want to give to this endpoint?

Reply:
it shall be the same as sourcemaps...no?

Reply:
The thing is that we don't have anymore the need of releases, thus `project:releases` shouldn't be needed. Do we have some docs on how permissions work and their meaning?